packages/bot-web-ui/src/stores/journal-store.js

```diff
@@ -106,5 +116,6 @@ export default class JournalStore {
     @action.bound
     clear() {
         this.unfiltered_messages.clear();
+        localStorage.setItem(journal_storage_key, JSON.stringify(this.unfiltered_messages));
```

Comment:
We could save a function call and just write an empty array here, e.g.:

```suggestion
        localStorage.setItem(journal_storage_key, '[]');
```